src/test/java/domain/profile/CustomerProfileServiceTest.java

```diff
@@ -0,0 +1,606 @@
+package domain.profile;
+
+import domain.listing.ListingDetails;
+import domain.listing.ListingRepository;
+import domain.listing.StarListing;
+
+import infrastructure.result.CustomerStarResult;
+import infrastructure.result.CustomerUnStarResult;
+import infrastructure.result.DeleteCustomerResult;
+import infrastructure.result.UpdateCustomerProfileResult;
+import infrastructure.sql.entity.CustomerProfileEntity;
+import infrastructure.sql.entity.ListingEntity;
+import io.quarkus.test.InjectMock;
+import io.quarkus.test.junit.QuarkusTest;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.BadRequestException;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+@QuarkusTest
```

Comment:
could use a little bit formatting for the files committed, I've just added `format_schema.xml` to our wiki, please import it intellij, select the file and press `ctrl` + `alt` + `L` to format the file, also remove extra spaces where's not needed.